feat: UIG-2706 - vl-input-field-component - introductie component

ksvlaanderen opened the pull request:
vl-input-field-component toegevoegd
Storybook en Cypress
Playground apps uitgebreid met form validatie voorbeeld

[Bamboo](https://www.milieuinfo.be/bamboo/browse/UIGOV-CUWC150)
[Jira](https://www.milieuinfo.be/jira/browse/UIG-2706)

Goldflow commented on libs/components/src/next/form/error-message/vl-error-message.component.ts:
```diff
@@ -13,7 +13,7 @@ export class VlErrorMessageComponent extends BaseLitElement {
 
     static get styles(): CSSResult[] {
-        return [resetStyle, formMessageStyle];
+        return [...commonStyles, formMessageStyle];
```
deze lijkt console errors te genereren

ksvlaanderen replied:
Fixed

ksvlaanderen committed:
feat: UIG-2706 - vl-input-field-component - introductie component

vl-input-field-component toegevoegd
Storybook en Cypress
Playground apps uitgebreid met form validatie voorbeeld
